integration/fuse/src/main/java/alluxio/fuse/cli/FuseShell.java

```diff
@@ -42,6 +47,24 @@
   public FuseShell(FileSystem fs, AlluxioConfiguration conf) {
     mFileSystem = fs;
     mConf = conf;
+    mCommands = loadCommands();
+  }
+
+  private Map<String, FuseCommand> loadCommands() {
```

Comment:
Why not directly use CommonUtils.loadCommands

Reply:
We must rename the alluxio.fuse.cli package to alluxio.cli package, and let the FuseCommand interface extends alluxio.cli.Command interface, already do this, thanks!